components/textfield/index.css

```diff
@@ -167,27 +189,27 @@ governing permissions and limitations under the License.
 
   .spectrum-Textfield.is-valid & {
     padding-inline-end: calc(
-      var(--spectrum-textfield-padding-x) +
+      var(--spectrum-textfield-padding-right) +
         var(--spectrum-icon-checkmark-medium-width) +
-        var(--spectrum-textfield-icon-margin-left)
+        var(--spectrum-textfield-success-icon-margin-left)
     );
   }
 
   .spectrum-Textfield.is-invalid & {
     padding-inline-end: calc(
-      var(--spectrum-textfield-padding-x) +
+      var(--spectrum-textfield-padding-right) +
         var(--spectrum-icon-alert-medium-width) +
-        var(--spectrum-textfield-icon-margin-left)
+        var(--spectrum-textfield-error-icon-margin-left)
     );
   }
 
   .spectrum-Textfield--multiline & {
```

Comment:
should this maybe be renamed into the classname `.spectrum-Textarea`?
Isn't a multiline textfield a textarea?

Reply:
You're right, but that's not how we currently treat this component and it is outside the scope of this PR. If you want to propose that, let's file an issue and have that discussion outside of this PR.